Description: Pass CFLAGS from dpkg
Author: Shengqi Chen <harry-chen@outlook.com>
Forwarded: not-needed
Last-Update: 2024-10-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -13,8 +13,9 @@
 
 LDLIBS ?= -lrt
 
+DPKG_CFLAGS := $(CFLAGS) $(CPPFLAGS)
 CFLAGS = $(CDEBUGFLAGS) $(DEFINES) $(EXTRA_DEFINES) \
-         $(SHA2_CFLAGS) $(BLAKE_CFLAGS)
+         $(SHA2_CFLAGS) $(BLAKE_CFLAGS) $(DPKG_CFLAGS)
 
 SRCS = babeld.c net.c kernel.c util.c interface.c source.c neighbour.c \
        route.c xroute.c message.c resend.c configuration.c local.c \
